src/libraries/Microsoft.Extensions.Configuration.Binder/src/buildTransitive/Microsoft.Extensions.Configuration.Binder.targets

```diff
@@ -8,6 +8,14 @@
         </ItemGroup>
     </Target>
 
+    <Target Name="_Microsoft_Extensions_Configuration_Binder_EnableInterceptors"
+            Condition="'$(EnableConfigurationBindingGenerator)' = 'true'"
+            AfterTargets="ResolvePackageDependenciesForBuild;ResolveNuGetPackageAssets">
+            
+        <!-- The configuration binding source generator uses a preview version of the compiler interceptors feature. Enable it implicitly when the generator is enabled. -->
+        <InterceptorsPreviewNamespaces>$(InterceptorsPreviewNamespaces);Microsoft.Extensions.Configuration.Binder.SourceGeneration</InterceptorsPreviewNamespaces>
```

Comment:
I'm surprised this is valid msbuild syntax. Is this a property or an item? I would expect these this element to be in a property group.

Reply:
Pushed stale commit.